lib/resources/messages.js

```diff
@@ -237,6 +232,13 @@ export class MessageInterface extends PlivoResourceInterface {
             })
         }
 
+        if (type === 'sms' && !text) {
```

Comment:
@aayush-plivo type is optional param, it will fail when arguments.length > 1
Please add the few change from here also https://github.com/plivo/plivo-node/pull/290 and close it